Imagine/Cache/Resolver/CacheResolver.php

```diff
@@ -230,10 +231,24 @@ protected function setDefaultOptions(OptionsResolverInterface $resolver)
             'index_key' => 'index',
         ));
 
-        $resolver->setAllowedTypes(array(
-            'global_prefix' => 'string',
-            'prefix' => 'string',
-            'index_key' => 'string',
-        ));
+        $allowedTypesList = array(
+          'global_prefix' => 'string',
+          'prefix' => 'string',
+          'index_key' => 'string',
+        );
+
+        if (version_compare(Kernel::VERSION_ID, '20600') >= 0) {
```

Comment:
@makasim Since symfony 2.6 setAllowedTypes api changed.

Now you call it with option and array of types instead off array where key is option and value is allowed types.